SpringProject/src/main/java/com/gradproject2019/Conference.java

```diff
@@ -1,14 +1,27 @@
+package com.gradproject2019;
+
+import javax.persistence.*;
 import java.time.Instant;
 
+@Entity
 public class Conference {
-    private long id;
+    @Id
+    private Long id;
+    @Column(name = "conference_name")
```

Comment:
Do the column names need to have the conference prefix if they are part of the conference object and conference table?

Reply:
Name, DateTime and Description are pre-determined types in SQL therefore it was our understanding that we could not use them as column names, this is why we re-named them without the prefix "conference" within the class

Reply:
I think it is fine to have the prefix as if you add more tables later requiring the column name they are easier to differentiate from each other.

Reply:
Did you try it? I've never heard of 'name' and 'description' being reserved keywords in SQL.